docs/bgp.md

```diff
@@ -0,0 +1,161 @@
+# Configuration of BGP peering
+
+By default, a Calico-Docker network uses a full BGP mesh to distribute routes 
```

Comment:
I don't really like the term Calico-Docker network. Could you should say By default, when running Calico on Docker, a full BGP mesh is used to distribute...?